src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1270,6 +1273,12 @@ public static void modifyAttributes()
         });
     }
 
+    public static void onBlockEntityBlocks(BiConsumer<BlockEntityType<?>, Block> callback)
```

Comment:
Also what is this name? Also, you fire the event on the mod event bus, why? You'll need to make the map concurrent if you do.

Reply:
It's here to prevent exposing some of the fields, but it can be moved to forge event factory and use getters. As for mod bus, I think that's a product of someone telling me to do something in discord, it was a long time ago. But I am pretty sure that can be forge bus.

Reply:
There is no reason to add getters.
But being here is fine. It doesn't really matter. The important part is that the function name is you know.. somewhat sane. 'onBlockEntityBlocks' What is the Block Entity Blocking?

And yes it shouldn't be on the mod bus.